src/components/BooksList.js

```diff
@@ -3,6 +3,8 @@ import { useSelector, useDispatch } from 'react-redux';
 import BookShow from './BookShow';
 import { getAllBooks } from '../redux/store';
```

Comment:
- [ ] I kindly suggest that you kindly follow the design given by not adding the text: "A lesson learned" to every book.

| Your Design  | Zeplin Design |
| ------------- | ------------- |
| <img width="255" alt="image" src="https://github.com/majdalkilany/Book-Store/assets/52934128/23ab85bf-5972-4941-ad30-6874374ccf38">  | <img width="193" alt="image" src="https://github.com/majdalkilany/Book-Store/assets/52934128/6cc35046-2737-432f-aff6-2c27cffb9d0f"> |

Reply:
Hello @topeogunleye   sorry but this point is not clear to me could you explain it please?

Reply:
> Hello @topeogunleye sorry but this point is not clear to me could you explain it please?

Whenever I add a new book, the word "ALessonLearned" is always added.

<img width="933" alt="image" src="https://github.com/majdalkilany/Book-Store/assets/52934128/55baa014-b139-443f-8cf0-99343aff5494">

Reply:
if you check the  Zeplin design you can find the text on it  

![new screen](https://github.com/majdalkilany/Book-Store/assets/60566840/729ba348-f5d2-460b-a7f7-d60985933b84)

Reply:
> if you check the Zeplin design you can find the text on it
> 
> ![new screen](https://user-images.githubusercontent.com/60566840/258602890-729ba348-f5d2-460b-a7f7-d60985933b84.png)

@majdalkilany, please kindly look carefully, a lesson learned is for just one book. In your case it is added on every  book

Reply:
> if you check the Zeplin design you can find the text on it
> 
> ![new screen](https://user-images.githubusercontent.com/60566840/258602890-729ba348-f5d2-460b-a7f7-d60985933b84.png)

I see your point, It's now optional

Reply:
@topeogunleye  I made it static because it doesn't require any specific functionality, so there's no strict need to add or remove it; it's just a block of text

Reply:
> @topeogunleye I made it static because it doesn't require any specific functionality, so there's no strict need to add or remove it; it's just a block of text

I agree 👍